source/appModules/mmc.py

```diff
@@ -20,10 +22,36 @@ def _get_focusRedirect(self):
 
 
 class MMCTableCell(IAccessible):
+	""" Cell and rowheader makes no sense for these controls. Mapping them to list items has added benefit
+	of suppressing selected. """
+	role = controlTypes.ROLE_LISTITEM
+
 	def event_selection(self):
 		if self.parent.hasFocus and api.getFocusObject() != self:
 			eventHandler.executeEvent("gainFocus", self)
 
+	def _get_positionInfo(self):
+		""" When 'Guess object position info when unavailable' is enabled
+		these controls reports very strange information such as 65537 of 12, especially in braille.
+		Disable reporting of position info all together. """
+		return None
+
+
+class emptyToolTip(ToolTip):

```

Comment:
You name this EmptyTooltip, but the code suggests that this tooltip can contain text. Therefore EmptyTooltip doesn't seem appropriate here.